.github/config/assertoor/network_params_blob.yaml

```diff
@@ -6,14 +6,11 @@ participants:
     validator_count: 32
     count: 2
   - el_type: ethrex
+    el_image: ethrex:latest
```

Comment:
This is now needed because the default is the deployed version: https://github.com/ethpandaops/ethereum-package/blob/82e5a7178138d892c0c31c3839c89d53ffd42d9a/src/package_io/input_parser.star#L17